docs/config-description.md

```diff
@@ -493,47 +493,39 @@ This document provides a list of all available Clio configuration properties in
 - **Type**: string
 - **Default value**: `none`
 - **Constraints**: The value must be one of the following: `int`, `uint`, `null`, `none`, `uuid`.
-- **Description**: The style for log tags.
+- **Description**: Log tags are unique identifiers for log messages. `uint`/`int` starts logging from 0 and increments, making it faster. In contrast, `uuid` generates a random unique identifier, which adds overhead.
 
 ### extractor_threads
 
 - **Required**: True
 - **Type**: int
 - **Default value**: `1`
 - **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
-- **Description**: The number of extractor threads.
+- **Description**: Number of threads used to extract data from ETL source.
 
 ### read_only
 
 - **Required**: True
 - **Type**: boolean
 - **Default value**: `True`
 - **Constraints**: None
-- **Description**: Indicates if the server should have read-only privileges.
-
-### txn_threshold
-
-- **Required**: True
-- **Type**: int
-- **Default value**: `0`
-- **Constraints**: The minimum value is `0`. The maximum value is `65535`.
-- **Description**: The transaction threshold value.
+- **Description**: Indicates if the server is allowed to write data to the database.
 
 ### start_sequence
 
 - **Required**: False
 - **Type**: int
 - **Default value**: None
 - **Constraints**: The minimum value is `0`. The maximum value is `4294967295`.
-- **Description**: Starting ledger index.
+- **Description**: If specified, the starting ledger where `Clio` will starts writing to database from.
```

Comment:
Typo: `starts` -> `start`. Maybe `If specified, the ledger index Clio will start writing to the database from.`. Also, Clio is spelled without ticks (unlike `rippled`).